packages/wallet/src/redux/protocols/indirect-funding/player-b/readme.md

```diff
@@ -0,0 +1,21 @@
+# Indirect Funding Protocol for Player B
```

Comment:
Nit: the state machine is missing a `start` state. In my opinion, the `start` diagram states are not necessary when the state machine has a single entry point. But we do have the `start` states in majority(?) of other diagrams.